docs/AutocompleteInput.md

```diff
@@ -97,6 +97,25 @@ When used inside a `<ReferenceInput>`, `<AutocompleteInput>` doesn't need a `cho
 See [Using in a `ReferenceInput>`](#using-in-a-referenceinput) below for more information.
 
 
+## `emptyValue`
+
+An empty choice is always added (with a default `''` value, which you can overwrite with the `emptyValue` prop) on top of the options. You can furthermore customize the empty choice by using the `emptyText` prop, which can receive string or a React Element, which doesn't receive any props.
+
+```jsx
+<AutocompleteInput
+    source="author_id"
+    emptyValue=""
+    emptyText="No author"
+    choices={[
+        { id: 123, name: 'Leo Tolstoi' },
+        { id: 456, name: 'Jane Austen' },
+    ]}
+/>
+```
+
+**Note**: `emptyValue` can not be set to `undefined` or `null` since the `dataProvider` method will recibe an empty string on submit due to the nature of HTML inputs.
```

Comment:
```suggestion
**Note**: `emptyValue` cannot be set to `undefined` or `null` since the `dataProvider` method will receive an empty string on submit due to the nature of HTML inputs.
```